editing/include/tests.js

```diff
@@ -5582,7 +5582,7 @@ function runConformanceTest(browserTest) {
 
             assert_equals(document.execCommand(browserTest[1][i][0], false, browserTest[1][i][1]),
                 expectedExecCommandReturnValues[i]);
-        }, testName + ": execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
+        }, testName + ": command #" + (i+1) + " execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
```

Comment:
Sorry, missed this bit. Sequence numbers are generally not a great idea as they have a tendency to change. It would be better to either only use this iff there would be a dupe, or to ensure that there is never a dupe.